torch/csrc/jit/operator.h

```diff
@@ -150,6 +150,15 @@ struct TORCH_API Operator {
   }
 
   c10::AliasAnalysisKind aliasAnalysisKind() const {
+    if (isC10Op()) {
```

Comment:
Oh cool, didn't think of putting it here but this should catch all call sites except for registration, exactly what we wanted.

Reply:
Will update the diff